### PR TITLE
Schools

### PR DESCRIPTION
**Pull request by kelseylam**
Created schools controller, model, added to routes. Also made associations between schools and users and schools and checkins. @charlesx2013 @sameerav

### CODE REVIEW

File: app/models/school.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  name       :string           default(""), not null
+#  lat        :decimal(10, 7)   default(0.0)
+#  lng        :decimal(10, 7)   default(0.0)
+#
+
+class School < ActiveRecord::Base
+  has_many :users
+  has_many :check_ins
```
Comment (amillman) [inline app/models/school.rb]: a school should also have a director, right?

File: app/models/user.rb [hunk 2]
```diff
@@ -29,6 +29,7 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  belongs_to :school
```
Comment (charlesx2013) [inline app/models/user.rb]: can you move this to below the has_many?

File: spec/factories/schools.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#  lat        :decimal(10, 7)   default(0.0)
+#  lng        :decimal(10, 7)   default(0.0)
+#
+
+FactoryGirl.define do
+  factory :school do
+    
+  end
+
+end
```
Comment (charlesx2013) [inline spec/factories/schools.rb]: delete this file!

File: spec/models/school_spec.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+#  name       :string           default(""), not null
+#  lat        :decimal(10, 7)   default(0.0)
+#  lng        :decimal(10, 7)   default(0.0)
+#
+
+require 'rails_helper'
+
+RSpec.describe School, :type => :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (charlesx2013) [inline spec/models/school_spec.rb]: this one too!

**Comment by charlesx2013** [comment]
LGTM

**Pull request merged**